Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/Ride.hs

```diff
@@ -97,7 +97,7 @@ initializeRide merchant driver booking mbOtpCode enableFrequentLocationUpdates m
   QRide.createRide ride
   QRideD.create rideDetails
   Redis.withWaitOnLockRedisWithExpiry (isOnRideWithAdvRideConditionKey driver.id.getId) 4 4 $ do
-    QDI.updateOnRide True (cast driver.id)
+    when (not booking.isScheduled) $ QDI.updateOnRide True (cast driver.id)
```

Comment:
What if schedule ride is within 30 minutes?

Reply:
isScheduled is false in case of booking under 30 mins .